.github/workflows/lint-sample.yml

```diff
@@ -22,12 +29,9 @@ jobs:
       - name: Run linter
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.59
-          working-directory: testdata/project-v4
-          args: --config .golangci.yml ./...
-      - name: Run linter
-        uses: golangci/golangci-lint-action@v6
-        with:
-          version: v1.59
-          working-directory: testdata/project-v4-with-plugins
+          version: v1.61.0
+          working-directory: ${{ matrix.folder }}
           args: --config .golangci.yml ./...
```

Comment:
I think we need to call make lint instead to ensure that it uses the lint setup configured in the makefile.
Same target that will be used by end users

Reply:
will make the change for `lint-sample` action

Reply:
done